### ISSUE TITLE
Documentation improvement, dangers of having an "old/wrong" fingerprint

### ISSUE THREAD

**Issue by strazzere** [Magisk-Modules-Repo/MagiskHidePropsConf#69]
Hey there,

Wanted to make a note as I did not see this listed in the documentation, I apologize if this was covered over on XDA - I don't frequent there anymore. Anyway, I've noticed complains from folks who sometimes use this module / data from this module that they cannot find certain applications (or find the wrong ones) via Google Play. It may be adding some language in the `README` around this to explain what is happening. If you'd like me to submit a PR, I can do so.

Basically, when the fingerprint is set, Google Play reads this value, which is obviously why the "certified device" piece suddenly passes. Though this also has a cascading effect of making Google Play believe you are that device for filtering as well. As an example, with the way bundles are produced for Youtube, if a device is presenting as a certified 10 device, but then upgrades to 11, Google Play will continue to believe it is on 10. This resulting in an older version of Youtube being shown as the most up to date APK the device can download. Specifically with Youtube, it will then barf and say "no I was made for an older version" as they do not rely on the fingerprint. It will then prompt a user to update the app and points them to the Play Store, which gets confused and is unable to access the page as it is filtered.

TLDR - 
If you mask your fingerprint as a device, you will likely be filtered to the hardware features of that specific device at it's specific version you're presenting. While this seems obvious when explained, it appears many people get confused by this.

Again, let me know if you'd like me to distill the above into something more concrete and submit a PR to the README/Issues section.

Thanks!

**Comment by Didgeridoohan** [comment]
That's a good idea. The docs can always be improved...

I'll look at that later, no need for an RP (unless you really want to, but I can't promise I'll use it).

**Comment by strazzere** [comment]
No problem, if I have free time this weekend (maybe not so likely?) and you haven't done it, I'll draft one.

Also, no worries if you don't use it -- that is why I posted it as an issue and offered to do it! :laughing: 

Thanks for this repo btw

**Comment by Didgeridoohan** [comment]
Ok...

I added some warnings and info on what might happen if you change props and use different/old device fingerprints.

https://github.com/Magisk-Modules-Repo/MagiskHide-Props-Config/blob/master/README.md#warning-1
https://github.com/Magisk-Modules-Repo/MagiskHide-Props-Config/blob/master/README.md#can-i-use-any-fingerprint
https://github.com/Magisk-Modules-Repo/MagiskHidePropsConf/blob/master/README.md#the-play-store-is-broken

I'm considering it a first draft, but hardly anyone reads the docs anyway...

If you've got anything to add I'm all ears.

**Comment by strazzere** [comment]
Generally speaking, I believe this is all good.

Also, I'm not 100% certain exactly how the filtering is done, as it's done server side for Google Play. The fingerprint is potentially a factor - but may not be the leading factor. As google states themselves, one "should not part the fingerprint themselves".

I think your changes to the README are good - I just wanted to state the above incase someone stumbles into this issue, etc.

**Comment by Didgeridoohan** [comment]
Yeah... It's because of that kind of uncertainty that I usually try to keep the docs a bit vague on subjects like this (might, possibly, could, etc).

Thanks for the inspiration and the kick in the behind.
